calibration/DOSTAN/CGINS-DOSTAN-01520__20180119.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+1520,CC_calibration_date,1516320000000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
Cal date of filename should be 20180129 (not 20180119).  milliseconds since 1970-01-01: 1517184000000

Reply:
Agreed.  The cal cert file is dated 2018-01-29